digBusinessTypeExtractor/documentation_classification.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+# @Author: ZwEin
+# @Date:   2016-10-06 23:36:45
+# @Last Modified by:   ZwEin
+# @Last Modified time: 2016-10-12 16:14:07
+# -*- coding: utf-8 -*-
+# @Author: ZwEin
+# @Date:   2016-09-23 12:58:37
+# @Last Modified by:   ZwEin
+# @Last Modified time: 2016-09-24 20:29:20
+
+import os
+import sys
+import csv
+import numpy as np
+from sklearn.externals import joblib
+from sklearn.neighbors import NearestNeighbors
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfVectorizer
+from os.path import expanduser
+import shutil
+
+
+##################################################################
+# Constant
+##################################################################
+
+STOP_WORDS = set([u'all', u'just', u'being', u'over', u'both', u'through', u'yourselves', u'its', u'before', u'o', u'hadn', u'herself', u'll', u'had', u'should', u'to', u'only', u'won', u'under', u'ours', u'has', u'do', u'them', u'his', u'very', u'they', u'not', u'during', u'now', u'him', u'nor', u'd', u'did', u'didn', u'this', u'she', u'each', u'further', u'where', u'few', u'because', u'doing', u'some', u'hasn', u'are', u'our', u'ourselves', u'out', u'what', u'for', u'while', u're', u'does', u'above', u'between', u'mustn', u't', u'be', u'we', u'who', u'were', u'here', u'shouldn', u'hers', u'by', u'on', u'about', u'couldn', u'of', u'against', u's', u'isn', u'or', u'own', u'into', u'yourself', u'down', u'mightn', u'wasn', u'your', u'from', u'her', u'their', u'aren', u'there', u'been', u'whom', u'too', u'wouldn', u'themselves', u'weren', u'was', u'until', u'more', u'himself', u'that', u'but', u'don', u'with', u'than', u'those', u'he', u'me', u'myself', u'ma', u'these', u'up', u'will', u'below', u'ain', u'can', u'theirs', u'my', u'and', u've', u'then', u'is', u'am', u'it', u'doesn', u'an', u'as', u'itself', u'at', u'have', u'in', u'any', u'if', u'again', u'no', u'when', u'same', u'how', u'other', u'which', u'you', u'shan', u'needn', u'haven', u'after', u'most', u'such', u'why', u'a', u'off', u'i', u'm', u'yours', u'so', u'y', u'the', u'having', u'once'])
+
+DC_CATEGORY_NAMES = [
+    'unknown',
+    'others',
+    'massage',
+    'escort',
+    'job_ads'
+]
+DC_CATEGORY_DICT = {
+    'unknown': -1,
+    'others': 1,
+    'massage': 2,
+    'escort': 3,
+    'job_ads': 4
+}
+
+USER_HOME_PATH = expanduser("~")
+
+APP_DIR = os.path.join(USER_HOME_PATH, 'WEDC_DATA')
```

Comment:
I don't think it's a good idea to create a default directory, especially not in the user's home.  You may not always have access to it.  When we're running on the cluster, some users don't even have a home directory.  The user should always specify where it is or at least make it an environment variable so we can override it.  I don't think having init_env() is a good idea.  if we're going to ship the package with the data, we might as well include a default tfidf/cosine model in the package as well, so we don't have to train the model everytime we run the program on a new machine.